### PR TITLE
PR to add Phoenix proposal for 2022

### PR DESCRIPTION
**Pull request by EdwardMoyse**
I wasn't quite sure whether we need to specify if we expect tasks to be short or long? 

Please let me know if I should rework this and I will do so tomorrow.

### CODE REVIEW

**Review by agheata — APPROVED** [review]
Fixed missing mentor and added Pittsburg

File: _gsocproposals/2022/proposal_Phoenix.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+[Phoenix](https://hepsoftwarefoundation.org/phoenix/) is a framework that can be used by any typical (e.g. colliding beam) High Energy Physics experiment. It was initially based on work done for the TrackML [Kaggle](https://www.kaggle.com/c/trackml-particle-identification)/[Codalab](https://competitions.codalab.org/competitions/20112) challenges (and internal use by ATLAS). 
+
+In the recent Advanced Computing and Analysis Techniques (ACAT) 2021 [conference](https://indico.cern.ch/event/855454/), two submissions cited Phoenix:
+* Open Data Detector (ACTS) [link](https://indico.cern.ch/event/855454/contributions/4596738/attachments/2352091/4013627/740_poster.pdf)
+* New Web Based Event Data and Geometry Visualisation for LHCb [link](https://indico.cern.ch/event/855454/contributions/4598428/attachments/2352310/4013170/572_Andreas-Pappas_ACAT2021_poster.pdf)
```
Comment (agheata) [inline _gsocproposals/2022/proposal_Phoenix.md:22]:
```suggestion
* New Web Based Event Data and Geometry Visualisation for LHCb [link](https://indico.cern.ch/event/855454/contributions/4598428/attachments/2352310/4018397/572_AndreasPappas_ACAT-2021_poster.pdf)
```

File: _gsocproposals/2022/proposal_Phoenix.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+### 3D menu for AR/VR
+Phoenix currently supports (in a quite basic form) VR and AR and even with the limited functionality we have right now, this is a very exciting way to use the event display and it is something we would really like to improve.
+
+However one current omission is there is no way to interact with the detector and event data visualisation: at the moment in AR/VR the user has no way to change what is shown,
+
+Phoenix’s menu system was always intended to be extensible, and so it should be possible to add a 3D menu when in VR mode. The major complication here will be that visualising detectors and event data on a phone is a very different experience to using it in a dedicated headset (such as a Meta Quest). Much care will need to be taken to make the UI as intuitive as possible. Once this is made to work, we can investigate adding a simple menu for AR mode (AR is more difficult, since it is less stable).
+
+
+#### Possible sub-tasks
+* Investigate available open source menu packages (e.g. https://discourse.threejs.org/t/three-mesh-ui), and evaluate the feasibility of incorporating them in Phoenix.
```
Comment (agheata) [inline _gsocproposals/2022/proposal_Phoenix.md:38]:
```suggestion
* Investigate available open source menu packages (e.g. https://github.com/felixmariotto/three-mesh-ui), and evaluate the feasibility of incorporating them in Phoenix.
```

**Review by agheata — APPROVED** [review]

**Comment by EdwardMoyse** [comment]
Thanks!

**Pull request merged**